mithril-common/src/entities/certificate_metadata.rs

```diff
@@ -97,6 +117,7 @@ impl CertificateMetadata {
     /// Computes the hash of the certificate metadata
     pub fn compute_hash(&self) -> String {
```

Comment:
As the goal is to remove it as soon as possible and as it's already marked as deprecated I think it's better to not include it in the hash in order to avoid a certificate hash re-computation when we remove it.

Reply:
I'd rather have the certificate hashes recomputed when the field is removed. All the fields of the certificate must be used to compute the hash.